src/ucp/core/ucp_worker.c

```diff
@@ -2188,6 +2188,10 @@ static void ucp_worker_put_flush_req(ucp_request_t *req)
 {
     ucp_ep_h ucp_ep = req->send.ep;
 
+#if UCS_ENABLE_ASSERT
+    ucs_assert(ucp_ep_ext_gen(ucp_ep)->num_discard_inprog > 0);
+    --ucp_ep_ext_gen(ucp_ep)->num_discard_inprog;
```

Comment:
maybe define some common func, like `ucp_ep_cnt_inc` and `ucp_ep_cnt_dec` and use it for all counters, `ref_cnt`, `num_discard_inprog` , `num_flush_inprog`?

Reply:
done